tests/cluster_tests/test_qsubs.sh

```diff
@@ -47,146 +46,5 @@ wait_lines ()
-wait_lines 'InternalParallelScikit/*.csv' 2 paralROM
-cd ..
-echo ''
-
-# third test (PostProcessor in parallel)
-cd InternalParallel/
-rm -Rf InternalParallelPostProcessorLS/*.csv
-../../../raven_framework test_internal_parallel_PP_LS.xml ../pbspro_mpi.xml ../cluster_runinfo_legacy.xml
-wait_lines 'InternalParallelPostProcessorLS/*.csv' 4 parallelPP
-cd ..
-echo ''
-
-# forth test (Topology Picard in parallel)
-cd InternalParallel/
-rm -Rf InternalParallelMSR/*.csv
-../../../raven_framework test_internal_MSR.xml ../pbspro_mpi.xml ../cluster_runinfo_legacy.xml
-wait_lines 'InternalParallelMSR/*.csv' 1 parallelMSR
-cd ..
-echo ''
-
-# fifth test (Ensamble Model Picard in parallel)
-cd InternalParallel/
-rm -Rf metaModelNonLinearParallel/*.png
-../../../raven_framework test_ensemble_model_picard_parallel.xml ../pbspro_mpi.xml ../cluster_runinfo_legacy.xml
-wait_lines 'metaModelNonLinearParallel/*.png' 3 parallelEnsemblePicard
-cd ..
-echo ''
-
-# sixth test (Ensamble Model Picard in parallel)
-cd InternalParallel/
-rm -Rf metaModelLinearParallel/*.png
-../../../raven_framework test_ensemble_model_linear_internal_parallel.xml ../pbspro_mpi.xml ../cluster_runinfo_legacy.xml
-wait_lines 'metaModelLinearParallel/*.png' 2 parallelEnsembleLinear
-cd ..
-echo ''
-
-# seven test (HybridModel Code in parallel)
-cd InternalParallel/
-rm -Rf hybridModelCode/*.csv
-../../../raven_framework test_hybrid_model_code.xml ../pbspro_mpi.xml ../cluster_runinfo_legacy.xml
-wait_lines 'hybridModelCode/*.csv' 1 parallelHybridModelCode
-cd ..
-echo ''
-
-# eighth test (HybridModel External Model in parallel)
-cd InternalParallel/
-rm -Rf hybridModelExternal/*.csv
-../../../raven_framework test_hybrid_model_external.xml ../pbspro_mpi.xml ../cluster_runinfo_legacy.xml
-wait_lines 'hybridModelExternal/*.csv' 1 parallelHybridModelExternal
-cd ..
-echo ''
-
-############################################
-# test parallel for internal Objects ENDED #
-############################################
-
-################################
-# other parallel objects tests #
-################################
-
-# Adaptive Sobol
-cd AdaptiveSobol/
-rm -Rf workdir/*
-../../../raven_framework test_adapt_sobol_parallel.xml ../pbspro_mpi.xml ../cluster_runinfo_legacy.xml
-wait_lines 'workdir/*.csv' 1 adaptiveSobol
-cd ..
-echo ''
-
-# Raven-Running-Raven (RAVEN code interface)
-cd RavenRunsRaven/raven_running_raven_internal_models/
-rm -Rf FirstMRun DatabaseStorage *csv testPointSet_dump.xml
-cd ..
-../../../raven_framework test_raven_running_raven_int_models.xml ../pbspro_mpi.xml ../cluster_runinfo_legacy.xml
-wait_lines 'raven_running_raven_internal_models/testP*.csv' 17 ravenRunningRaven
-cd ..
-echo ''
-
-
-if test $num_fails -eq 0; then
-    echo ALL PASSED
-else
-    echo FAILED: $num_fails $fails
-fi
-exit $num_fails
+cd ${RAVEN_FRAMEWORK_DIR}/..
+./run_tests --only-run-types="qsub" --re=cluster_tests
```

Comment:
Doesn't need to be on this PR, but could we potentially parallelize these qsub tests?